src/App.tsx

```diff
@@ -6,16 +6,23 @@ import AcceptMember from 'page/Admin';
 import AuthRoute from 'components/common/AuthRoute';
 import DuesManagement from 'page/DuesManagement';
 import DefaultLayout from 'layout/DefaultLayout';
-import { Suspense } from 'react';
+import { Suspense, useEffect } from 'react';
 import MyPage from 'page/MyPage';
 import DuesSetup from 'page/DuesSetup';
 import EditDues from 'page/EditDues';
 import LoadingSpinner from 'layout/LoadingSpinner';
 import TrackInfo from 'page/Track';
 import FindPassword from 'page/FindPassword';
 import Role from 'page/Role';
+import { useLoginState } from 'store/loginStore';
 
 function App() {
+  const { setMe } = useLoginState();
+
+  useEffect(() => {
+    setMe();
+    /* eslint-disable react-hooks/exhaustive-deps */
+  }, []);
```

Comment:
약간 편견일 수 있긴한데, useEffect에 의존성 배열을 비워둬도 되나요?

Reply:
페이지 마운트될 때 한 번만 로그인 정보를 저장하면 될 것이다라고 판단했습니다
더 좋은 방법이 있을까요?

Reply:
```suggestion
  useEffect(
    () => {
      setMe();
    },
    [setMe],
  );
```
물론 의도상 빈배열을 쓰는게 맞긴하지만 setMe를 의존성 배열에 넣어 관리해주는게 lint룰상 더 적절할 것 같기도 합니다